run_clang_tidy.sh

```diff
@@ -4,7 +4,7 @@ set -uo pipefail
 
 FILE=$1
 
-EXCLUDED_DIRECTORY=("extensions/pdh" "extensions/windows-event-log" "nanofi")
+EXCLUDED_DIRECTORY=("extensions/pdh" "extensions/windows-event-log" "extensions/smb" "nanofi")
```

Comment:
Why is smb excluded from clang-tidy?

Reply:
I've excluded it (like the other windows only extensions), because we the CI doesnt care if the extension is being compiled or not, and thus the ubuntu-clang ci will try and fail to find the windows headers etc...

We could (and probably should) run the clang-tidy on windows aswell, but I think thats probably out of the scope of this PR.

Reply:
My bad, at this point somehow I just forgot that this was Windows-only :smile: